src/jenkins_job_insight/reportportal.py

```
@@ -398,27 +406,46 @@ def push_classifications(
+                response_body = ""
                 if exc.response is not None:
+                    response_body = exc.response.text
                     try:
                         rp_body = exc.response.json()
-                        detail = rp_body.get("message", "")
+                        detail = (
+                            rp_body.get("message", response_body)
+                            if isinstance(rp_body, dict)
+                            else response_body
+                        )
                     except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)